ProjFS.Mac/PrjFSKext/PrjFSKext/VirtualizationRoots.cpp

```diff
@@ -352,50 +352,60 @@ VirtualizationRootResult VirtualizationRoot_RegisterProviderForPath(PrjFSProvide
         }
         else
         {
-            FsidInode vnodeIds = Vnode_GetFsidAndInode(virtualizationRootVNode, vfsContext);
-            uint32_t rootVid = vnode_vid(virtualizationRootVNode);
+            char virtualizationRootCanonicalPath[PrjFSMaxPath] = "";
+            int virtualizationRootCanonicalPathLength = sizeof(virtualizationRootCanonicalPath);
+            err = vn_getpath(virtualizationRootVNode, virtualizationRootCanonicalPath, &virtualizationRootCanonicalPathLength);
```

Comment:
Super minor:  Update `// Return values:` comment to include that this function can return error value(s) returned by `vn_getpath`.

Also, do you know if our call to `vnode_lookup` will help prevent `vn_getpath` failures?  (I wasn't sure if there was any relationship between the name cache and `vnode_lookup`).

Reply:
It might reduce the likelihood, but as we aren't locking the name cache, nothing can be guaranteed anyway.